File: BikeShop.Backend/BikeShop.Products/BikeShop.Products.Domain/Entities/ProductFilter.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BikeShop.Products.Domain.Entities
{
    public class ProductFilter : BaseEntity
    {
        public string Name { get; set; }
        public string GroupName { get; set; }
        public bool IsCollapsed { get; set; }
        public bool IsRetailVisible { get; set; }
        public bool IsB2BVisible { get; set; }
        public int SortOrder { get; set; }
        public int GroupSortOrder { get; set; }
    }
}

File: BikeShop.Frontend/webclient/src/entities/DataTransferObjects/ProductFilter.ts
export interface ProductFilter {
    id: number;
    createdAt: string;
    updatedAt: string;
    enabled: boolean;
    name: string;
    groupName: string;
    isCollapsed: boolean;
    isRetailVisible: boolean;
    isB2BVisible: boolean;
    sortOrder: number;
    groupSortOrder: number;
}